src/main.c

```diff
@@ -5,8 +5,8 @@
 
 #include <cglm/cglm.h>
 
-#include "log/log.h"
-#include "log/proginfo.h"
+#include "util/log/log.h"
```

Comment:
please order alphabetically

Reply:
Hi @inventor02 ,

This diff shows that the lines have been removed

Reply:
Hi @a-caine,

Apologies, there is an error in this comment.

Please ignore this comment.

With kindest regards